polkit.py

```diff
@@ -58,7 +61,9 @@ def make_dotfiles(path_to_generate_files_sh: str):
         with open(local_path_to_dotpolicy, 'w') as f:
             f.write(dotpolicy_with_path)
             print("I need root permissions to move the file just this one time.")
-            os.system("sudo mv " + local_path_to_dotpolicy + " " + path_to_dotpolicy)
+            os.system("x-terminal-emulator -e sudo mv " + local_path_to_dotpolicy + " " + path_to_dotpolicy)
```

Comment:
The `print` statement is invisible to the user. We need to tell them what we are doing in the new terminal window.
```suggestion
            os.system("x-terminal-emulator -e echo 'I need root permissions to move the file just this one time.' && sudo mv " + local_path_to_dotpolicy + " " + path_to_dotpolicy)
```